http://bugs.gentoo.org/show_bug.cgi?id=259995

In file included from miniwget.c:31:
/usr/include/utility.h:221: error: expected declaration specifiers or
‘...’ before ‘OPTIONS’


--- transmission-1.42/third-party/miniupnp/miniwget.c
+++ transmission-1.42/third-party/miniupnp/miniwget.c
@@ -26,9 +26,9 @@
 #include <arpa/inet.h>
 #define closesocket close
 #endif
-/* for MIN() macro : */
+
 #if defined(__sun) || defined(sun)
-#include <utility.h>
+#define MIN(x,y) (((x)<(y))?(x):(y))
 #endif
 
 /* miniwget2() :
